src/Microsoft.TestPlatform.ObjectModel/Client/Interfaces/ITestDiscoveryEventsHandler2.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Client.Interfaces
```

Comment:
Interface `ITestDiscoveryEventsHander` existed under `Microsoft.VisualStudio.TestPlatform.ObjectModel.Client` namespace. 

Should we use the same namespace as used by v1 interface?